CHANGELOG.md

```diff
@@ -39,6 +38,7 @@ the detailed section referring to by linking pull requests or issues.
 * Implement Asset service for Data Management API (#931)
 * Implement ContractDefinition service for Data Management API (#940)
 * Implement ContractNegotiation service for Data Management API (#957)
+* Add client to delete data transfer to Data Plane (#988)
```

Comment:
```suggestion
* Let consumer delegate data transfer to Data Plane (#988)
```

Reply:
This is not related to the consumer. This PR is about offering the possibility to the Control Plane to delegate the data transfers to its Data Plane(s).